Source/ORTS.Settings/UserSettings.cs

```diff
@@ -309,6 +309,10 @@ public enum DirectXFeature
         public bool PerformanceTuner { get; set; }
         [Default(true)]
         public bool SuppressShapeWarnings { get; set; }
+        [Default(1)]
+        public int VfsLogLevel { get; set; }
+        [Default(false)]
+        public bool VfsNoAutoMount { get; set; }
```

Comment:
In addition to removing the negative in the name, please can we have this default to **not** auto-mounting?

We want the first version of this code to behave as close to the old code as possible by default, but allow for experimentation with the additional capabilities the VFS offers.